packages/@react-spectrum/actionbar/docs/ActionBar.mdx

```diff
@@ -0,0 +1,240 @@
+          )}
+        </TableHeader>
+        <TableBody items={rows}>
+          {item => (
+            <Row>
+              {columnKey => <Cell>{item[columnKey]}</Cell>}
+            </Row>
+          )}
+        </TableBody>
+      </TableView>
+      <ActionBar
+        items={barItems}
+        isEmphasized
+        selectedItemCount={selectedKeys === 'all' ? selectedKeys : selectedKeys.size}
+        onClearSelection={() => {
+          setSelectedKeys(new Set());
+        }}
+        onAction={key => alert(`Performing ${key} action on ${selectedKeys.size} rows.`)}
+        {...otherProps}>
+        {item => (
+          <Item key={item.key}>
+            {item.label}
+          </Item>
+        )}
+      </ActionBar>
+    </ActionBarContainer>
+  );
+}
+
+<DynamicActionBar />
+```
+
+### Layout
+ActionBarContainer accepts an ActionBar alongside any renderable children. It accepts the typical selection of [style props](#props), allowing you to customize its layout.
+The example below applies a static height on the ActionBarContainer instead of matching the height of the TableView.
+
+```tsx example
+// Same example as above
+<DynamicActionBar containerHeight="size-5000" />
+```
+
+### Accessibility
+The contents of the ActionBar should follow the same accessibility guidelines as ActionGroup's items, see the [ActionGroup docs](./ActionGroup.html#accessibility) for more details.
+
+### Internationalization
+To internationalize an ActionBar, the content provided to all child items should be localized. A localized string for the selection count and clear button `aria-label` will be provided
+automatically. For languages that are read right-to-left (e.g. Hebrew and Arabic), the layout of the ActionBar is automatically flipped.
+
+## Events
+Use the `onClearSelection` prop as a callback to handle when the clear button in the ActionBar is pressed. Additionally, the `onAction` prop can be used as a callback for when the user
+presses any of the buttons in the ActionBar's ActionGroup. The `key` from the pressed button will be passed into the callback.
+
+The following example clears row selection within the TableView when the clear button is pressed and alerts the user as to what
+action is being taken on the selected rows when the ActionButtons are pressed.
+
+```tsx example
+// Same example as above
+<DynamicActionBar />
+```
+
+## Props
+
+### ActionBarContainer props
+<PropTable component={docs.exports.ActionBarContainer} links={docs.links} style={{marginBottom: '40px'}} />
+
+### ActionBar props
+<PropTable component={docs.exports.ActionBar} links={docs.links} />
+
+## Visual options
+
+### isEmphasized
+`isEmphasized` should be applied to the ActionBar when it should call attention, such as when floating in a TableView. It should be omitted if the ActionBar should blend in
+with the rest of the UI, directing a user’s focus to elsewhere in a view.
+
+```tsx example
+<ActionBarContainer>
+  <ActionBar>
+    <Item key="edit">
```

Comment:
A bit strange because it isn't a "realistic" example (no accompanying element other than the ActionBars) but just trying to not make it take up so much vertical space (aka CardView and TableView are very tall code wise and physically)

Reply:
Updating all to include list or table